### DISCUSSION TITLE
[@types/react-mentions] Out of date definitions

### DISCUSSION THREAD

**Discussion by agbares** [DefinitelyTyped/DefinitelyTyped#61394] [Issues with a @types package]
The type definitions were made for [react-mentions v4.1](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8638abd6fc735c4a48039aba84dbc1433e031a06/types/react-mentions/index.d.ts#L1). The package is now at v4.4.7. 

The missing type definitions are for:
- `forceSuggestionsAboveCursor` (introduced in [v4.2.0](https://github.com/signavio/react-mentions/releases/tag/v4.2.0))
- `onAdd` callback param changes ([v4.3.0](https://github.com/signavio/react-mentions/releases/tag/v4.3.0))
- `customSuggestionsContainer` ([v4.4.0](https://github.com/signavio/react-mentions/releases/tag/v4.4.0))

I also don't mind updating the type defs in a few weeks, although I haven't made nor modified type defs before.

[a comment by an automated account is not shown]

**Comment by activescott** [comment]
@agbares You're most welcome to add those type definitions. I'll be happy to review when you upload them if you ping me.
